src/components/LabeledSwitch/LabeledSwitch.jsx

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash';
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition }  from '../../util/component-definition';
+import Switch from '../Switch/Switch';
+
+const boundClassNames = lucidClassNames.bind('&-LabeledSwitch');
+const {
+	bool,
+	func,
+	node,
+	object,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"], "extend": "Switch"}
+ *
+ * This is a composite of the `Switch` component and the native `label`
+ * element.
+ */
+const LabeledSwitch = React.createClass(createLucidComponentDefinition({
+	displayName: 'LabeledSwitch',
+
+	childProps: {
+		Label: {
```

Comment:
I've been following @ogupte's convention of documenting these at the top level as well.

https://github.com/appnexus/lucid/pull/115/files#diff-c276da651c3910657398754600e5a6d6R127